docs/r2v/data_processing_output.md

```diff
@@ -0,0 +1,289 @@
+object detection but correlates the objects temporally. For a given object, 
+the tracking JSON looks like:
+
+```json
+{
+        "_type": "TrackedObject",
+        "class_label": 3,
+        "instance_label": 1006,
+        "path": [
+            {
+                "_type": "Frame",
+                "timestamp": 1593178665.992904,
+                "index": 0,
+                "views": []
+            },
+            {
+                "_type": "Frame",
+                "timestamp": 1593178666.093764,
+                "index": 1,
+                "views": [
+                    {
+                        "_type": "View",
+                        "camera": 0,
+                        "bbox": {
+                            "_type": "BoundingBox",
+                            "y": 210.0,
+                            "x": 428.0,
+                            "height": 9.0,
+                            "width": 16.0
+                        },
+                        "mask": {
+                            "_type": "RLEMask",
+                            "size": [
+                                422,
+                                906
+                            ],
+                            "counts": [
+                                180829,
+                                3,
+                                ...
+                            ]
+                        }
+                    }
+                ]
+            },
+            {
+                "_type": "Frame",
+                "timestamp": 1593178667.093764,
+                "index": 2,
+                ...
+```
+
+For a single tracked object, the JSON defines:
+
+* `class_label`: The label from the segmentation for this type of object
+* `instance_label`: The unique label for this particular object
+* `path`: The list of frames that 
+
+For a single frame, the JSON defines:
+
+* `timestamp`: The UTC timestamp for the time the frame was acquired
+* `index`: The index into the data set the frame was acquired
+* `views`: The array defines the frames and RLE Maks in the frame that the 
+object was tracked to
+ * `camera`: The index of the camera the image was taken from
+ * `bbox`: The bounding box within the frame of the object
+ * `mask`: The RLE encoded mask of the object to the pixel level
+
+## Mesh
+
+The mesh data generated by the Real-to-Virtual pipeline includes textured and
+untextured meshes of the different assets produced by the data processing. 
+The textured meshes that are generated from the stitched point cloud data can
+be found in the `mesh/mesh_parts` directory. These have been processed through
+several meshing algorithms for optimal triangulation for each type of object.
+
+The `mesh/tiles` directory contains the output from the monoDrive Direct Texture
+algorithm. Here, each mesh is organized into an individual tile `.obj` file that
+has been textured by project the imagery directly onto the mesh. When the
```

Comment:
should it be "by projecting the imagery"?

Reply:
Yes. Fixed.